### ISSUE TITLE
Remove emoji from browser devtools tabs

### ISSUE THREAD

**Issue by inoyakaigor** [facebook/react#16509]
![image](https://user-images.githubusercontent.com/1235286/63173784-425c4680-c049-11e9-9708-5fd2c99f9e43.png)
Make it possible to remove emoji from the tabs. It violate the common devtools style. 

Also in Firefox 68.0.2 @ Macos it is duplicated
<img width="292" alt="изображение" src="https://user-images.githubusercontent.com/1235286/63266594-ef78ce00-c298-11e9-90ca-73f95ef25181.png">

**Comment by bvaughn** [comment]
What is the "common devtools style" you speak of?

**Comment by gaearon** [comment]
I assume the way other (built-in) tabs are labeled.

**Comment by inoyakaigor** [comment]
@bvaughn  I mean, the devtools have a strict interface without emojis anywhere

**Comment by rafde** [comment]
> ... strict interface without emojis anywhere

If that were the case then emoji's wouldn't work.

**Comment by bvaughn** [comment]
I agree that Firefox's double display isn't ideal. I quite like the emoji icon in Chrome though. It makes the tab names smaller, which saves space for other tab names.

**Comment by gaearon** [comment]
Can we turn them off just for FF?

**Comment by gaearon** [comment]
Maybe we could also get Chrome to consider FF-like behavior with the icon. At least as an opt in. That would solve the problem both for us and possibly for other extensions that would like shorter titles.

**Comment by bvaughn** [comment]
> Can we turn them off just for FF?

Yes, certainly. Seems like a good course of action for Firefox.

**Comment by szyszak** [comment]
On separate note: this emoji is so small I couldn't recognize what it represents until I saw this issue. I thought is was a tiny cogwheel, usually representing "options". Old "React Devtools" name was much more readable and user friendly.

**Comment by bvaughn** [comment]
> Old "React Devtools" name was much more readable and user friendly.

Using "React Components" and "React Profiler" for tab names takes up a lot of space. That's what I did initially for the new DevTools, but given all of the other (important) tabs the browser has, it felt like an inefficient use of the horizontal space.

**Comment by NateRadebaugh** [comment]
Was there a discussion on why two tabs were created instead of using just one? My dev tools tabs list is already getting pretty long and I imagine the profiler will be used less frequently than the components tab.

**Comment by gaearon** [comment]
@NateRadebaugh That's exactly the reason.

Previously, Profiler was _always_ taking up vertical space in the React tab. Whether you use it or not. (Maybe you didn't see it on an older version of React, but eventually you'd see it showing up.)

Now, you can drag the tabs you use to the left (Chrome DevTools lets you do that), and move the tabs you don't care about (e.g. the Profiler) to the end.

**Comment by bvaughn** [comment]
Dan pretty much summed it up. The choice of splitting into two top-level tabs was to make most efficient use of space. Old DevTools _always_ wasted the top ~25 pixels with a tab bar that only had "Elements" and "Profiler" tabs:
<img width="598" alt="Screen Shot 2019-08-21 at 12 12 57 PM" src="https://user-images.githubusercontent.com/29597/63461238-0b04e580-c40d-11e9-8f7a-8dd6485ac4d6.png">

New DevTools inlines those tabs into the browser's own tab bar, and as a result lets you reorder and/or hide the ones you care about less.

**Comment by hristo-kanchev** [comment]
I've removed the ⚛️emoji from the panes in Firefox. Gonna wait for https://github.com/facebook/react/pull/16381 to get merged before opening a PR.
